<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {background-color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>76</width>
        <height>19</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>直播间ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEditLiveNumber">
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>8206033</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ButtonConnect">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;楷体&quot;;
</string>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="ButtonDisconnect">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>断开</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
